pkg/cli/cc/cc.go

```diff
@@ -16,10 +19,25 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+var (
+	ErrNotRunning = errors.New("topaz is not running, use 'topaz start' or 'topaz run' to start")
+	ErrIsRunning  = errors.New("topaz is already running, use 'topaz stop' to stop")
+	ErrNotServing = errors.New("topaz gRPC endpoint not SERVING")
+)
+
 type CommonCtx struct {
 	Context context.Context
 	UI      *clui.UI
-	NoCheck bool
+	Config  *CLIConfig
+}
+
+type CLIConfig struct {
+	NoCheck           bool
+	TopazConfigFile   string
+	ContainerName     string
+	ContainerRegistry string
+	ContainerImage    string
+	ContainerTag      string
```

Comment:
Should `ContainerPlatform` be part of this list as it is a settable parameter like the others? 
Besides consistency with the other parameters, I can see this being useful when insisting to use amd64 images on a Mac arm64 machine.